<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AxesDialog</class>
 <widget class="QDialog" name="AxesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>224</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot Limits</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <item row="1" column="1">
     <widget class="QLineEdit" name="editXMax"/>
    </item>
    <item row="1" column="2">
     <widget class="QCheckBox" name="LogX">
      <property name="text">
       <string>Log X</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="xMax">
      <property name="text">
       <string>x Max</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="autoX">
      <property name="text">
       <string>Auto X</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="xMin">
      <property name="text">
       <string>x Min</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="editXMin"/>
    </item>
    <item row="2" column="2">
     <widget class="QCheckBox" name="autoY">
      <property name="text">
       <string>Auto Y</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="editYMin"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="yMin">
      <property name="text">
       <string>y Min</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="yMax">
      <property name="text">
       <string>y Max</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="editYMax"/>
    </item>
    <item row="3" column="2">
     <widget class="QCheckBox" name="LogY">
      <property name="text">
       <string>Log Y</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>editXMin</tabstop>
  <tabstop>editXMax</tabstop>
  <tabstop>editYMin</tabstop>
  <tabstop>editYMax</tabstop>
  <tabstop>autoX</tabstop>
  <tabstop>LogX</tabstop>
  <tabstop>autoY</tabstop>
  <tabstop>LogY</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
